atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -509,7 +509,8 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
+                    'autorebuild': is_rebuild(self.workflow)
```

Comment:
Should this be done in get_build instead? Just thinking if this is a property of the build or the archive.